cmd/tether/main_linux.go

```
@@ -38,6 +38,10 @@ func main() {
 		halt()
 	}()
 
+	logFile, _ := os.OpenFile("/dev/ttyS1", os.O_WRONLY|os.O_SYNC, 0644)
```

do not ignore the error here